src/python-common/ceph/deployment/service_spec.py

```diff
@@ -55,6 +55,16 @@ class CertificateSource(Enum):
     CEPHADM_SIGNED = "cephadm-signed"
 
 
+class MonitorCertSource(Enum):
+    """
+    - REUSE_SERVICE_CERT: Use Service cert for monitoring
+    """
+    INLINE = CertificateSource.INLINE.value
+    REFERENCE = CertificateSource.REFERENCE.value
+    CEPHADM_SIGNED = CertificateSource.CEPHADM_SIGNED.value
+    REUSE_SERVICE_CERT = "reuse_service_cert"
```

Comment:
I'm wondering if this new field is really needed! I mean, if the user sets `monitor_ssl` in the spec and doesn't provide `monitor_ssl_cert` and `monitor_ssl_key` then we can automatically fallback to the service certs without adding this new option, wdyt?